src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {
+    private static final String INVALID_FILE_PATH = "";
+    private static final String EMPTY_FILE_PATH = "src/test/resources/empty-file.csv";
+    private static final String EXAMPLE_FILE_PATH = "src/test/resources/example-file.csv";
```

Comment:
```suggestion
    private static final String VALID_FILE_PATH = "src/test/resources/example-file.csv";
```
for naming consistency